docker-compose.yml

```diff
@@ -38,12 +40,49 @@ services:
       - "3306"
     volumes:
       - wca_db_volume:/var/lib/mysql
+      - ./docker/my.cnf:/etc/my.cnf
     # Suppress unneeded messages via https://stackoverflow.com/a/55706057/2558618
     cap_add:
       - SYS_NICE
+    networks:
+      - wca-main
+
+  nginx:
```

Comment:
Change it so the user has to specify a flag in order to boot nginx and php

Reply:
I added an overwrite file for this